README.md

```diff
@@ -40,19 +40,19 @@ Enter the URL to the appropriate endpoint to the "Source type" used, enter a tit
 
 Queue any scheduled harvest jobs.
 
-    $ docker-compose run --rm app ckan harvest harvest_run
+    $ docker-compose run -d --no-deps app ckan --plugin=ckanext-harvest harvester run
```

Comment:
Do you mind dropping the `-d` and keeping `--rm`? I prefer to keep commands like this in the foreground so you can see the logs. Also, in case you don't have anything running, dropping `--no-deps` will allow the dependent containers to start properly.

```suggestion
    $ docker-compose run --rm app ckan --plugin=ckanext-harvest harvester run
```